docs/modules/plugins/pages/plugin-db.adoc

```diff
@@ -45,3 +45,33 @@ a|The strategy defining how records with duplicate keys should be handled:
 |The maximum time to wait for DB query completion
 
 |===
+
+
+== Table Transformers
+
+NOTE: The table transformers properties marked with *bold* are mandatory.
+
+=== FROM_DB
+
+`FROM_DB` transformer generates table from DB via SQL queries.
+
+[cols="1,3", options="header"]
+|===
+|Parameter
+|Description
+
+|[subs=+quotes]`*dbKey*`
+|Database key
+
+|`*sqlQuery*`
+|SQL select query to get data
+
+|`nullReplacement`
+|Value to replace NULLs in result data set
+|===
+
+.The transformer based on the default configuration
```

Comment:
```suggestion
`FROM_DB` transformer generates table based on the data retirved from the
specified database using SQL query.

[cols="1,3", options="header"]
|===
|Parameter
|Description

|`*dbKey*`
|Database key

|`*sqlQuery*`
|SQL `SELECT` query to retrieve the data from DB

|`nullReplacement`
|The value to put in the generating table instead of `NULL`-s contained in the DB data set
|===

.Generate ExampleTable using data from H2 database
```